src/ui/src/js/controllers/admin_garden.js

```diff
@@ -19,13 +19,15 @@ export default function adminGardenController(
     GardenService,
     EventService) {
   $scope.setWindowTitle('gardens');
-
+  $scope.gardenCreateSchema = GardenService.CreateSCHEMA;
+   $scope.gardenCreateForm = GardenService.CreateFORM;
   $scope.successCallback = function(response) {
     $scope.response = response;
     $scope.data = response.data;
 
   };
-
+  $scope.garden_name = null;
```

Comment:
This is too generic for this page. The who page is to manage Gardens and having a field called `garden_name` gives no uniqueness to what the value is actually doing. Give this a more meaningful name.